src/nodedef.cpp

```diff
@@ -272,7 +272,8 @@ void TextureSettings::readSettings()
 	connected_glass                = g_settings->getBool("connected_glass");
 	translucent_liquids            = g_settings->getBool("translucent_liquids");
 	enable_minimap                 = g_settings->getBool("enable_minimap");
-	node_texture_size              = std::max<u16>(g_settings->getU16("texture_min_size"), 1);
+	node_texture_size              = rangelim(g_settings->getU16("texture_min_size"),
```

Comment:
Why enforce a min-limit in the code? The default is already set to 192. If a user wants this lower, why not let them?

Reply:
the idea is: the user should not be able to ruin the artistic choice of the game author

Reply:
I see. One could then argue why have a setting like this at all? I.e. the server should be in control.
Having a good default, but letting a user set whatever they want makes more sense to me (it's already in the "advanced" section).

Reply:
I think larger sizes help with moire even more and make it less blurry (obviously), so it sort of still makes sense.